packages/office-ui-fabric-react/src/components/Facepile/Facepile.styles.ts

```diff
@@ -1,11 +1,6 @@
 import { IFacepileStyleProps, IFacepileStyles } from './Facepile.types';
 
-import {
-  IStyle,
-  hiddenContentStyle,
-  getFocusStyle,
-  getGlobalClassNames,
-} from '../../Styling';
+import { IStyle, hiddenContentStyle, getFocusStyle, getGlobalClassNames } from '../../Styling';
```

Comment:
I'm not sure why FacePile got dragged into this PR. Looks to just be Prettier formatting with no functional changes.